AndroidXCI/ftlModelBuilder/src/test/kotlin/dev/androidx/ci/codegen/SchemaProcessorTest.kt

```diff
@@ -18,6 +18,7 @@ package dev.androidx.ci.codegen
 
 import com.google.common.truth.Truth.assertThat
 import org.junit.Test
+import java.util.TreeMap
 
 class SchemaProcessorTest {
```

Comment:
can we add a single test with swapped inputs to make sure that the output is the same?

Reply:
added a test that was failing before this change.